src/index.tsx

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
 import { Provider } from "react-redux";
 import { AppThemeProvider } from "./themes/ThemesProvider";
 import CssBaseline from "@mui/material/CssBaseline";
-
+import { setupStore } from "./store";
```

Comment:
```suggestion
import { store } from "./store";
```
Если это уже стор, то зачем его сетапить называя с глагола